src/linux/component.rs

```diff
@@ -176,14 +421,18 @@ pub(crate) fn get_components() -> Vec<Component> {
         }
         components.sort_by(|c1, c2| c1.label.to_lowercase().cmp(&c2.label.to_lowercase()));
     }
-    if is_file("/sys/class/thermal/thermal_zone0/temp") {
-        // Specfic to raspberry pi.
-        components.push(Component::new(
-            "CPU".to_owned(),
-            Path::new("/sys/class/thermal/thermal_zone0/temp").to_path_buf(),
-            None,
-            None,
-        ));
+
+    // Specfic to raspberry pi tries to avoid it, better info from hwmon.
+    // If we already have something containing `Core` or `CPU` ignore it.
+    if is_file("/sys/class/thermal/thermal_zone0/temp")
+        && components
+            .iter()
+            .position(|c| c.label.contains("Core") || c.label.contains("CPU"))
```

Comment:
I'm really feeling uneasy about this check: it's so easy to have a component with either "CPU" or "Core" inside it. Or the opposite: we already have this information but it's named "CORE" or "Cpu". Is there really no other way? If there is no better way, maybe it would be better to have a duplicated entry instead...

Reply:
we get rrrrrriiiiid of raspberry pi special case!

Reply:
The duplicated entry is due to the fact that motherboard can report something general and cpu also withhhh noooo standard name.